src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -343,22 +343,24 @@ def test_process_declarations(f2008_parser):
     reader = FortranStringReader("integer, parameter :: i1 = 1")
     fparser2spec = Specification_Part(reader).content[0]
     processor.process_declarations(fake_parent, [fparser2spec], [])
-    assert fake_parent.symbol_table.lookup("i1").constant_value == 1
+    newsymbol = fake_parent.symbol_table.lookup("i1")
+    assert newsymbol.is_constant
+    assert isinstance(newsymbol.constant_value, Literal)
+    assert newsymbol.constant_value.value == "1"
 
     reader = FortranStringReader("real, parameter :: i2 = 2.2, i3 = 3.3")
     fparser2spec = Specification_Part(reader).content[0]
     processor.process_declarations(fake_parent, [fparser2spec], [])
-    assert fake_parent.symbol_table.lookup("i2").constant_value == 2.2
-    assert fake_parent.symbol_table.lookup("i3").constant_value == 3.3
+    assert fake_parent.symbol_table.lookup("i2").constant_value.value == "2.2"
+    assert fake_parent.symbol_table.lookup("i3").constant_value.value == "3.3"
 
     # Static constant expresions are not supported
-    reader = FortranStringReader("real, parameter :: a = 1.1, b = a * 2")
+    reader = FortranStringReader("real, parameter :: i4 = 1.1, i5 = i4 * 2")
```

Comment:
You need to update the comment above this line :-)

Reply:
Ups.. done